site/en/docs/webstore/cws-dashboard-distribution/index.md

```diff
@@ -90,13 +90,35 @@ the Chrome Web Store. You can either:
 When you publish to the world, your item will be immediately visible to the region(s) you select,
 and it'll be visible in the store's search results.
 
+## Managing a test version and a production version
```

Comment:
@awfuchs Thoughts on this title? Suggestions are welcome 😁

Reply:
I think "Publishing a test version" works.

Reply:
Great!